core/domain/platform_parameter_domain.py

```diff
@@ -29,14 +29,15 @@
 import utils
 
 
-SERVER_MODES = utils.create_enum('dev', 'test', 'prod') # pylint: disable=invalid-name
+SERVER_MODES = python_utils.create_enum('dev', 'test', 'prod') # pylint: disable=invalid-name
 FEATURE_STAGES = SERVER_MODES # pylint: disable=invalid-name
-DATA_TYPES = utils.create_enum('bool', 'string', 'number') # pylint: disable=invalid-name
+DATA_TYPES = python_utils.create_enum('bool', 'string', 'number') # pylint: disable=invalid-name
 
 ALLOWED_SERVER_MODES = [
-    SERVER_MODES.dev, SERVER_MODES.test, SERVER_MODES.prod]
+    SERVER_MODES.dev.value, SERVER_MODES.test.value, SERVER_MODES.prod.value]
```

Comment:
Hmmm, should we have lists of enums and then transform to values only if needed?

Reply:
Basically, I think that in the code we can pass enums around but we should only make sure that when we actually save the values into the datastore they are always strings and not enums.

Reply:
Yeah if we are transforming it further down the road, it makes sense to make this array with just enums rather than value. Will try it out

Reply:
done